ui/src/app/views/recent-work/component.tsx

```diff
@@ -52,7 +53,9 @@ export const RecentWork = (fp.flow as any)(
             cohortsApi().getCohortsInWorkspace(namespace, id),
             conceptSetsApi().getConceptSetsInWorkspace(namespace, id)
           ]);
-           // TODO Remove this cast when we switch to fetch types
+          const notebookName = notebooks.map(notebook => notebook.name);
```

Comment:
Nit: I'd avoid the intermediate variable and just put this into the setState call.